hbase-server/src/main/java/org/apache/hadoop/hbase/master/balancer/BaseLoadBalancer.java

```diff
@@ -1223,6 +1225,20 @@ protected boolean areSomeRegionReplicasColocated(Cluster c) {
     return false;
   }
 
+  protected final synchronized boolean idleRegionExist(Cluster c){
+    boolean isServerExistsWithMoreRegions = false;
+    boolean isServerExistsWithZeroRegions = false;
+    for (int[] serverList: c.regionsPerServer){
+      if (serverList.length > 1) {
+        isServerExistsWithMoreRegions = true;
+      }
+      if (serverList.length < 1) {
```

Comment:
nit: this can be `serverList.length == 0`

Reply:
Thank you!